src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -122,18 +116,6 @@ class AuthScreens extends React.Component {
 
         App.fixAccountAndReloadData();
         App.setUpPoliciesAndNavigate(this.props.session);
-
-        // Refresh the personal details, timezone and betas every 30 minutes
-        // There is no pusher event that sends updated personal details data yet
-        // See https://github.com/Expensify/ReactNativeChat/issues/468
-        this.interval = Timers.register(setInterval(() => {
-            if (this.props.network.isOffline) {
-                return;
-            }
-            PersonalDetails.fetchPersonalDetails();
-            User.getBetas();
```

Comment:
I _think_ it's fine to get rid of the betas here as well. Unsure why we were also periodically refreshing these but regardless periodic refresh seems like not what we want to do longer term.

Reply:
Ah we were holding on removing User.getBetas from the timer until `AuthenticateUser` is implemented  - https://github.com/Expensify/App/pull/9769. I think it's probably ok to remove it now though.

Reply:
Works for me 👍